src/index.js

```diff
@@ -4,7 +4,16 @@ import './index.css';
 import App from './App.jsx';
 import registerServiceWorker from './registerServiceWorker';
 import data from './data.js';
+import { createStore } from 'redux'
+import { Provider } from 'react-redux'
+import reducer from './reducers/index.js'
 
+const store = createStore(reducer) ; 
```

Comment:
our store can have multiple reducers and its a good practice to have keys specifying each reducer.
the state should look something like:
```
{
  products: []
}
```
Please use 
`import {combineReducers} from 'redux';` for this